util/raclgen.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+r"""RACL Generator
+"""
+
+from io import StringIO
+import argparse
+from mako.template import Template
+import logging as log
+from reggen.ip_block import IpBlock
+from raclgen.lib import parse_racl_config, parse_racl_mapping
+
+
+def main():
+
+    parser = argparse.ArgumentParser(prog="raclgen")
+
+    parser.add_argument("--racl-config",
+                        "-r",
+                        required=True,
+                        help="Path to RACL config hjson file.")
+
+    parser.add_argument("--ip",
+                        "-i",
+                        required=True,
+                        help="Path to ip block hjson file.")
+
+    parser.add_argument("--mapping",
+                        "-m",
+                        required=True,
+                        help="Path to RACL mapping hjson file.")
+
+    parser.add_argument("--if-name",
+                        default=None,
+                        help='Interface name. Required if multiple bus_interfaces exist.')
+
+    args = parser.parse_args()
+
+    try:
+        ip_block = IpBlock.from_path(path = args.ip, param_defaults = [])
+    except ValueError as err:
+        log.error(str(err))
+        exit(1)
+
+    try:
+        parsed_racl_config = parse_racl_config(args.racl_config)
+    except ValueError as err:
+        log.error(str(err))
+        exit(1)
+
+    parsed_register_mapping, policy_names = parse_racl_mapping(
+        parsed_racl_config, args.mapping, args.if_name, ip_block)
+
+    template = """
+<% import math %>\
```

Comment:
Can we try to make a template fragment (file) and use that here and in the RACL pkg rendering?